Yank/experiment.py

```diff
@@ -502,7 +502,7 @@ class ExperimentBuilder(object):
         can load it later by using :func:`parse` (default is None).
     job_id : None or int
         If you want to split the experiments among different executions,
-        you can set this to an integer 0 <= job_id <= n_jobs-1, and this
+        you can set this to an integer 1 <= job_id <= n_jobs, and this
```

Comment:
I don't see it in the yaml docs, but just to make sure: `job_id` is not a parameter which can be YAML specified, correct?

Reply:
Correct! It's just a CLI option to make it work with array jobs.